spec.bs

```diff
@@ -2799,34 +2821,32 @@ CORP violation report=] algorithm, as leaving it unfenced may cause a privacy le
 
      1. Set |navigable| to |navigable|'s [=navigable/parent=].
 
-  1. If |navigable| is null, return null.
-
-  1. Otherwise, return |navigable|'s [=navigable/active document=]'s [=Document/origin=].
+  1. Return |navigable|.
 
   <div id="get-initiator-ancestor-root-origin-example" class="example">
-    Given a {{Document}} embedded inside of a <{fencedframe}>, this algorithm gets the origin of the
-    document's nearest fenced frame ancestor's nearest fenced frame ancestor. If no such "fenced
+    Given a {{Document}} embedded inside of a <{fencedframe}>, this algorithm gets the navigable of
+    the document's nearest fenced frame ancestor's nearest fenced frame ancestor. If no such "fenced
     frame grand-ancestor" exists, the algorithm returns null. Therefore, this algorithm only returns
-    an origin when given a Document that is, at a minimum, in a fenced frame which is itself in a
+    a navigable when given a Document that is, at a minimum, in a fenced frame which is itself in a
     fenced frame. For example, for the given frame tree structure:
 
     ```
-    Main frame (origin A)
-      <fencedframe> (origin B)
-        <iframe> (origin C)
-          <fencedframe> (origin D)
-            <iframe> (origin E)
-              <fencedframe> (origin F)
+    Main frame (A)
```

Comment:
Keeping "origin A" here (instead of just "A") seems fine? Not sure why it was removed.

Reply:
The algorithm isn't returning the origin anymore. I guess it's fine either way. I'll change it back to origin since the rest of the comments still makes it clear this is returning a navigable now.